src/views/Home/index.view.tsx

```diff
@@ -3,6 +3,7 @@ import Hero from "views/Home/components/Hero/index.view";
 import Button from "components/Button/index.view";
 import EmailSubscriptionForm from "components/EmailSubscription/index.view";
 import MLHBanner from "components/MLHBanner/index.view";
+import Schedule from "views/Home/components/Schedule/index.view";
```

Comment:
We can do the import here from `./components/Schedule/index.view;` instead of referencing back to `views`